gui/src/locales/en.json

```diff
@@ -164,5 +164,11 @@
     "usernameInvalid":"Invalid username.",
     "invalidRootUsername":"The username must not be 'root' or 'Root'.",
     "applyContinue":"Apply and continue"
+  },
+  "regex-validation": {
+    "bucket-input-tooltip":"The bucket name must be of minimum 4 characters and maximum 56 characters. Only lowercase, numbers, dash(-) and dot (.) are allowed. The bucket name cannot start and end with a dash (-) or dot(.)." ,
+    "account-tooltip":"The account name must be of minimum 4 characters and maximum 56 characters. The username must be alphanumeric and can contain underscore (_) and dash (-).",
```

Comment:
Please add a space after ":" or just format the file.

Reply:
Resolved the above comment